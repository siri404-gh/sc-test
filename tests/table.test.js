import React from 'react';
import { render, unmountComponentAtNode } from 'react-dom';
import { act } from 'react-dom/test-utils';
import cones from '../cones.json';

// jest.mock('../src/utils', () => ({
//     drawTable: jest.fn(),
//     calculateTimeSeries: jest.fn(),
// }));

import Table from '../src/Table';

describe('Table', () => {
    let container;
    const props = {
        cones,
        riskLevel: 10,
        initialSum: 1000,
    };

    beforeEach(() => {
        container = document.createElement('section');
        document.body.appendChild(container);
    });

    afterEach(() => {
        unmountComponentAtNode(container);
        container.remove();
        container = null;
    });

    it('Should render without props', () => {
        act(() => {
            render(<Table />, container);
        });
        expect(container).toMatchSnapshot();
    });

    it('Should render with props', () => {
        act(() => {
            render(<Table {...props} />, container);
        });
        expect(container).toMatchSnapshot();
    });
});