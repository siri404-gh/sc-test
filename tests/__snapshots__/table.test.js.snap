// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table Should render with props 1`] = `
<section>
  <table>
    <thead>
      <tr>
        <th>
          month
        </th>
        <th>
          good
        </th>
        <th>
          median
        </th>
        <th>
          bad
        </th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>
          0
        </td>
        <td>
          999.9999999999998
        </td>
        <td>
          1000
        </td>
        <td>
          999.9999999999998
        </td>
      </tr>
      <tr>
        <td>
          1
        </td>
        <td>
          1228.154591163118
        </td>
        <td>
          1202.6451589425214
        </td>
        <td>
          1177.6655713659943
        </td>
      </tr>
      <tr>
        <td>
          2
        </td>
        <td>
          1448.1814571876612
        </td>
        <td>
          1405.8263465393757
        </td>
        <td>
          1364.7100001282122
        </td>
      </tr>
      <tr>
        <td>
          3
        </td>
        <td>
          1669.1358181828764
        </td>
        <td>
          1609.5449806715635
        </td>
        <td>
          1552.0816320539725
        </td>
      </tr>
      <tr>
        <td>
          4
        </td>
        <td>
          1891.5640408333493
        </td>
        <td>
          1813.8024829705757
        </td>
        <td>
          1739.2376764472806
        </td>
      </tr>
      <tr>
        <td>
          5
        </td>
        <td>
          2115.5985982654147
        </td>
        <td>
          2018.6002788283781
        </td>
        <td>
          1926.0492463111395
        </td>
      </tr>
      <tr>
        <td>
          6
        </td>
        <td>
          2341.2695698600073
        </td>
        <td>
          2223.9397974072945
        </td>
        <td>
          2112.489858563247
        </td>
      </tr>
      <tr>
        <td>
          7
        </td>
        <td>
          2568.573313378157
        </td>
        <td>
          2429.8224716500263
        </td>
        <td>
          2298.566761939343
        </td>
      </tr>
      <tr>
        <td>
          8
        </td>
        <td>
          2797.493963661311
        </td>
        <td>
          2636.2497382896554
        </td>
        <td>
          2484.299438321749
        </td>
      </tr>
      <tr>
        <td>
          9
        </td>
        <td>
          3028.011466108351
        </td>
        <td>
          2843.2230378596114
        </td>
        <td>
          2669.7115692911193
        </td>
      </tr>
      <tr>
        <td>
          10
        </td>
        <td>
          3260.10488100898
        </td>
        <td>
          3050.743814703795
        </td>
        <td>
          2854.8277318222313
        </td>
      </tr>
      <tr>
        <td>
          11
        </td>
        <td>
          3493.75379751284
        </td>
        <td>
          3258.8135169865945
        </td>
        <td>
          3039.671984343803
        </td>
      </tr>
      <tr>
        <td>
          12
        </td>
        <td>
          3728.9389255696656
        </td>
        <td>
          3467.43359670306
        </td>
        <td>
          3224.267274827611
        </td>
      </tr>
      <tr>
        <td>
          13
        </td>
        <td>
          3965.6423112009957
        </td>
        <td>
          3676.605509688992
        </td>
        <td>
          3408.635225546022
        </td>
      </tr>
      <tr>
        <td>
          14
        </td>
        <td>
          4203.847374905959
        </td>
        <td>
          3886.330715631059
        </td>
        <td>
          3592.796094695348
        </td>
      </tr>
      <tr>
        <td>
          15
        </td>
        <td>
          4443.538867656634
        </td>
        <td>
          4096.610678077111
        </td>
        <td>
          3776.7688204302285
        </td>
      </tr>
      <tr>
        <td>
          16
        </td>
        <td>
          4684.70279072516
        </td>
        <td>
          4307.446864446262
        </td>
        <td>
          3960.5711010657133
        </td>
      </tr>
      <tr>
        <td>
          17
        </td>
        <td>
          4927.32630240268
        </td>
        <td>
          4518.840746039186
        </td>
        <td>
          4144.219488388008
        </td>
      </tr>
      <tr>
        <td>
          18
        </td>
        <td>
          5171.3976230989
        </td>
        <td>
          4730.793798048406
        </td>
        <td>
          4327.72948258465
        </td>
      </tr>
      <tr>
        <td>
          19
        </td>
        <td>
          5416.905944383401
        </td>
        <td>
          4943.307499568534
        </td>
        <td>
          4511.115623232789
        </td>
      </tr>
      <tr>
        <td>
          20
        </td>
        <td>
          5663.841344446662
        </td>
        <td>
          5156.383333606648
        </td>
        <td>
          4694.391573868135
        </td>
      </tr>
      <tr>
        <td>
          21
        </td>
        <td>
          5912.194710849468
        </td>
        <td>
          5370.022787092602
        </td>
        <td>
          4877.570199265383
        </td>
      </tr>
      <tr>
        <td>
          22
        </td>
        <td>
          6161.957670603011
        </td>
        <td>
          5584.227350889438
        </td>
        <td>
          5060.663635388134
        </td>
      </tr>
      <tr>
        <td>
          23
        </td>
        <td>
          6413.122527213749
        </td>
        <td>
          5798.998519803704
        </td>
        <td>
          5243.683352374016
        </td>
      </tr>
      <tr>
        <td>
          24
        </td>
        <td>
          6665.682204146079
        </td>
        <td>
          6014.3377925960285
        </td>
        <td>
          5426.640211102422
        </td>
      </tr>
      <tr>
        <td>
          25
        </td>
        <td>
          6919.630194095153
        </td>
        <td>
          6230.246671991468
        </td>
        <td>
          5609.54451395166
        </td>
      </tr>
      <tr>
        <td>
          26
        </td>
        <td>
          7174.960513466235
        </td>
        <td>
          6446.726664690001
        </td>
        <td>
          5792.406050350134
        </td>
      </tr>
      <tr>
        <td>
          27
        </td>
        <td>
          7431.66766149077
        </td>
        <td>
          6663.779281377091
        </td>
        <td>
          5975.234137690557
        </td>
      </tr>
      <tr>
        <td>
          28
        </td>
        <td>
          7689.746583458724
        </td>
        <td>
          6881.406036734213
        </td>
        <td>
          6158.037658128275
        </td>
      </tr>
      <tr>
        <td>
          29
        </td>
        <td>
          7949.192637599101
        </td>
        <td>
          7099.608449449411
        </td>
        <td>
          6340.825091731229
        </td>
      </tr>
      <tr>
        <td>
          30
        </td>
        <td>
          8210.001565192908
        </td>
        <td>
          7318.3880422278635
        </td>
        <td>
          6523.604546397595
        </td>
      </tr>
      <tr>
        <td>
          31
        </td>
        <td>
          8472.16946355138
        </td>
        <td>
          7537.746341802616
        </td>
        <td>
          6706.3837849085985
        </td>
      </tr>
      <tr>
        <td>
          32
        </td>
        <td>
          8735.692761536107
        </td>
        <td>
          7757.684878945094
        </td>
        <td>
          6889.170249438903
        </td>
      </tr>
      <tr>
        <td>
          33
        </td>
        <td>
          9000.568197338116
        </td>
        <td>
          7978.205188475891
        </td>
        <td>
          7071.97108380873
        </td>
      </tr>
      <tr>
        <td>
          34
        </td>
        <td>
          9266.792798266739
        </td>
        <td>
          8199.308809275464
        </td>
        <td>
          7254.793153725916
        </td>
      </tr>
      <tr>
        <td>
          35
        </td>
        <td>
          9534.36386233029
        </td>
        <td>
          8420.997284294815
        </td>
        <td>
          7437.643065236321
        </td>
      </tr>
      <tr>
        <td>
          36
        </td>
        <td>
          9803.278941417357
        </td>
        <td>
          8643.272160566316
        </td>
        <td>
          7620.527181573762
        </td>
      </tr>
      <tr>
        <td>
          37
        </td>
        <td>
          10073.53582591003
        </td>
        <td>
          8866.13498921449
        </td>
        <td>
          7803.451638578167
        </td>
      </tr>
      <tr>
        <td>
          38
        </td>
        <td>
          10345.132530581426
        </td>
        <td>
          9089.587325466811
        </td>
        <td>
          7986.422358829201
        </td>
      </tr>
      <tr>
        <td>
          39
        </td>
        <td>
          10618.067281647145
        </td>
        <td>
          9313.6307286646
        </td>
        <td>
          8169.445064626614
        </td>
      </tr>
      <tr>
        <td>
          40
        </td>
        <td>
          10892.338504854863
        </td>
        <td>
          9538.266762273866
        </td>
        <td>
          8352.525289931826
        </td>
      </tr>
      <tr>
        <td>
          41
        </td>
        <td>
          11167.944814511335
        </td>
        <td>
          9763.496993896246
        </td>
        <td>
          8535.668391372865
        </td>
      </tr>
      <tr>
        <td>
          42
        </td>
        <td>
          11444.885003355539
        </td>
        <td>
          9989.322995279937
        </td>
        <td>
          8718.879558402916
        </td>
      </tr>
      <tr>
        <td>
          43
        </td>
        <td>
          11723.158033198277
        </td>
        <td>
          10215.746342330638
        </td>
        <td>
          8902.163822692259
        </td>
      </tr>
      <tr>
        <td>
          44
        </td>
        <td>
          12002.763026257444
        </td>
        <td>
          10442.768615122579
        </td>
        <td>
          9085.526066825314
        </td>
      </tr>
      <tr>
        <td>
          45
        </td>
        <td>
          12283.699257124541
        </td>
        <td>
          10670.39139790956
        </td>
        <td>
          9268.971032365933
        </td>
      </tr>
      <tr>
        <td>
          46
        </td>
        <td>
          12565.966145306473
        </td>
        <td>
          10898.616279135938
        </td>
        <td>
          9452.503327347627
        </td>
      </tr>
      <tr>
        <td>
          47
        </td>
        <td>
          12849.563248291746
        </td>
        <td>
          11127.444851447814
        </td>
        <td>
          9636.127433239672
        </td>
      </tr>
      <tr>
        <td>
          48
        </td>
        <td>
          13134.49025509551
        </td>
        <td>
          11356.878711704034
        </td>
        <td>
          9819.847711434324
        </td>
      </tr>
      <tr>
        <td>
          49
        </td>
        <td>
          13420.74698024273
        </td>
        <td>
          11586.91946098742
        </td>
        <td>
          10003.668409295999
        </td>
      </tr>
      <tr>
        <td>
          50
        </td>
        <td>
          13708.333358152628
        </td>
        <td>
          11817.568704615924
        </td>
        <td>
          10187.593665809281
        </td>
      </tr>
      <tr>
        <td>
          51
        </td>
        <td>
          13997.249437891736
        </td>
        <td>
          12048.828052153807
        </td>
        <td>
          10371.627516858369
        </td>
      </tr>
      <tr>
        <td>
          52
        </td>
        <td>
          14287.495378265206
        </td>
        <td>
          12280.699117422862
        </td>
        <td>
          10555.773900168562
        </td>
      </tr>
      <tr>
        <td>
          53
        </td>
        <td>
          14579.071443219693
        </td>
        <td>
          12513.183518513722
        </td>
        <td>
          10740.036659935857
        </td>
      </tr>
      <tr>
        <td>
          54
        </td>
        <td>
          14871.977997533435
        </td>
        <td>
          12746.282877797135
        </td>
        <td>
          10924.419551170005
        </td>
      </tr>
      <tr>
        <td>
          55
        </td>
        <td>
          15166.215502771094
        </td>
        <td>
          12979.998821935245
        </td>
        <td>
          11108.926243772337
        </td>
      </tr>
      <tr>
        <td>
          56
        </td>
        <td>
          15461.78451348343
        </td>
        <td>
          13214.33298189301
        </td>
        <td>
          11293.560326369156
        </td>
      </tr>
      <tr>
        <td>
          57
        </td>
        <td>
          15758.685673633505
        </td>
        <td>
          13449.28699294952
        </td>
        <td>
          11478.325309918722
        </td>
      </tr>
      <tr>
        <td>
          58
        </td>
        <td>
          16056.919713232746
        </td>
        <td>
          13684.862494709456
        </td>
        <td>
          11663.224631108349
        </td>
      </tr>
      <tr>
        <td>
          59
        </td>
        <td>
          16356.487445171306
        </td>
        <td>
          13921.061131114522
        </td>
        <td>
          11848.261655557297
        </td>
      </tr>
      <tr>
        <td>
          60
        </td>
        <td>
          16657.38976222938
        </td>
        <td>
          14157.884550454863
        </td>
        <td>
          12033.439680838754
        </td>
      </tr>
      <tr>
        <td>
          61
        </td>
        <td>
          16959.62763425638
        </td>
        <td>
          14395.334405380685
        </td>
        <td>
          12218.761939334468
        </td>
      </tr>
      <tr>
        <td>
          62
        </td>
        <td>
          17263.202105506232
        </td>
        <td>
          14633.412352913661
        </td>
        <td>
          12404.231600932566
        </td>
      </tr>
      <tr>
        <td>
          63
        </td>
        <td>
          17568.114292118575
        </td>
        <td>
          14872.120054458584
        </td>
        <td>
          12589.851775580442
        </td>
      </tr>
      <tr>
        <td>
          64
        </td>
        <td>
          17874.36537973537
        </td>
        <td>
          15111.459175814889
        </td>
        <td>
          12775.625515701586
        </td>
      </tr>
      <tr>
        <td>
          65
        </td>
        <td>
          18181.956621244528
        </td>
        <td>
          15351.431387188346
        </td>
        <td>
          12961.55581848598
        </td>
      </tr>
      <tr>
        <td>
          66
        </td>
        <td>
          18490.88933464189
        </td>
        <td>
          15592.038363202668
        </td>
        <td>
          13147.645628062033
        </td>
      </tr>
      <tr>
        <td>
          67
        </td>
        <td>
          18801.16490100407
        </td>
        <td>
          15833.281782911225
        </td>
        <td>
          13333.897837557935
        </td>
      </tr>
      <tr>
        <td>
          68
        </td>
        <td>
          19112.784762564595
        </td>
        <td>
          16075.16332980875
        </td>
        <td>
          13520.31529105934
        </td>
      </tr>
      <tr>
        <td>
          69
        </td>
        <td>
          19425.75042088785
        </td>
        <td>
          16317.6846918431
        </td>
        <td>
          13706.90078546996
        </td>
      </tr>
      <tr>
        <td>
          70
        </td>
        <td>
          19740.063435133492
        </td>
        <td>
          16560.847561426974
        </td>
        <td>
          13893.6570722812
        </td>
      </tr>
      <tr>
        <td>
          71
        </td>
        <td>
          20055.72542040708
        </td>
        <td>
          16804.6536354498
        </td>
        <td>
          14080.586859256271
        </td>
      </tr>
      <tr>
        <td>
          72
        </td>
        <td>
          20372.738046190596
        </td>
        <td>
          17049.1046152896
        </td>
        <td>
          14267.692812034216
        </td>
      </tr>
      <tr>
        <td>
          73
        </td>
        <td>
          20691.10303484844
        </td>
        <td>
          17294.202206824706
        </td>
        <td>
          14454.97755565794
        </td>
      </tr>
      <tr>
        <td>
          74
        </td>
        <td>
          21010.82216020514
        </td>
        <td>
          17539.94812044586
        </td>
        <td>
          14642.443676032155
        </td>
      </tr>
      <tr>
        <td>
          75
        </td>
        <td>
          21331.897246189365
        </td>
        <td>
          17786.34407106803
        </td>
        <td>
          14830.093721313473
        </td>
      </tr>
      <tr>
        <td>
          76
        </td>
        <td>
          21654.330165541294
        </td>
        <td>
          18033.39177814237
        </td>
        <td>
          15017.930203238137
        </td>
      </tr>
      <tr>
        <td>
          77
        </td>
        <td>
          21978.122838579526
        </td>
        <td>
          18281.092965668322
        </td>
        <td>
          15205.955598390286
        </td>
      </tr>
      <tr>
        <td>
          78
        </td>
        <td>
          22303.277232024353
        </td>
        <td>
          18529.44936220552
        </td>
        <td>
          15394.172349413751
        </td>
      </tr>
      <tr>
        <td>
          79
        </td>
        <td>
          22629.795357873474
        </td>
        <td>
          18778.46270088595
        </td>
        <td>
          15582.582866171446
        </td>
      </tr>
      <tr>
        <td>
          80
        </td>
        <td>
          22957.679272328445
        </td>
        <td>
          19028.134719426005
        </td>
        <td>
          15771.189526854247
        </td>
      </tr>
      <tr>
        <td>
          81
        </td>
        <td>
          23286.93107476851
        </td>
        <td>
          19278.46716013861
        </td>
        <td>
          15959.994679042857
        </td>
      </tr>
      <tr>
        <td>
          82
        </td>
        <td>
          23617.552906768648
        </td>
        <td>
          19529.46176994535
        </td>
        <td>
          16149.00064072473
        </td>
      </tr>
      <tr>
        <td>
          83
        </td>
        <td>
          23949.546951161083
        </td>
        <td>
          19781.12030038875
        </td>
        <td>
          16338.209701268768
        </td>
      </tr>
      <tr>
        <td>
          84
        </td>
        <td>
          24282.9154311362
        </td>
        <td>
          20033.444507644417
        </td>
        <td>
          16527.62412235975
        </td>
      </tr>
      <tr>
        <td>
          85
        </td>
        <td>
          24617.66060938233
        </td>
        <td>
          20286.43615253332
        </td>
        <td>
          16717.24613889446
        </td>
      </tr>
      <tr>
        <td>
          86
        </td>
        <td>
          24953.78478726123
        </td>
        <td>
          20540.09700053408
        </td>
        <td>
          16907.07795984215
        </td>
      </tr>
      <tr>
        <td>
          87
        </td>
        <td>
          25291.290304018243
        </td>
        <td>
          20794.4288217953
        </td>
        <td>
          17097.121769070483
        </td>
      </tr>
      <tr>
        <td>
          88
        </td>
        <td>
          25630.17953602506
        </td>
        <td>
          21049.433391147904
        </td>
        <td>
          17287.37972613859
        </td>
      </tr>
      <tr>
        <td>
          89
        </td>
        <td>
          25970.454896053463
        </td>
        <td>
          21305.112488117502
        </td>
        <td>
          17477.853967059964
        </td>
      </tr>
      <tr>
        <td>
          90
        </td>
        <td>
          26312.118832578686
        </td>
        <td>
          21561.467896936876
        </td>
        <td>
          17668.54660503514
        </td>
      </tr>
      <tr>
        <td>
          91
        </td>
        <td>
          26655.17382911077
        </td>
        <td>
          21818.50140655834
        </td>
        <td>
          17859.45973115681
        </td>
      </tr>
      <tr>
        <td>
          92
        </td>
        <td>
          26999.622403552778
        </td>
        <td>
          22076.214810666323
        </td>
        <td>
          18050.59541508821
        </td>
      </tr>
      <tr>
        <td>
          93
        </td>
        <td>
          27345.467107584212
        </td>
        <td>
          22334.609907689784
        </td>
        <td>
          18241.95570571638
        </td>
      </tr>
      <tr>
        <td>
          94
        </td>
        <td>
          27692.710526069026
        </td>
        <td>
          22593.68850081484
        </td>
        <td>
          18433.542631780623
        </td>
      </tr>
      <tr>
        <td>
          95
        </td>
        <td>
          28041.355276486578
        </td>
        <td>
          22853.452397997295
        </td>
        <td>
          18625.35820247869
        </td>
      </tr>
      <tr>
        <td>
          96
        </td>
        <td>
          28391.404008384783
        </td>
        <td>
          23113.903411975356
        </td>
        <td>
          18817.404408050657
        </td>
      </tr>
      <tr>
        <td>
          97
        </td>
        <td>
          28742.859402853916
        </td>
        <td>
          23375.043360282107
        </td>
        <td>
          19009.68322034158
        </td>
      </tr>
      <tr>
        <td>
          98
        </td>
        <td>
          29095.724172021248
        </td>
        <td>
          23636.874065258384
        </td>
        <td>
          19202.196593344757
        </td>
      </tr>
      <tr>
        <td>
          99
        </td>
        <td>
          29450.001058564078
        </td>
        <td>
          23899.39735406535
        </td>
        <td>
          19394.946463725355
        </td>
      </tr>
      <tr>
        <td>
          100
        </td>
        <td>
          29805.692835242007
        </td>
        <td>
          24162.61505869734
        </td>
        <td>
          19587.934751326087
        </td>
      </tr>
      <tr>
        <td>
          101
        </td>
        <td>
          30162.802304445635
        </td>
        <td>
          24426.529015994554
        </td>
        <td>
          19781.163359655235
        </td>
      </tr>
      <tr>
        <td>
          102
        </td>
        <td>
          30521.332297763198
        </td>
        <td>
          24691.141067655968
        </td>
        <td>
          19974.63417635825
        </td>
      </tr>
      <tr>
        <td>
          103
        </td>
        <td>
          30881.285675562376
        </td>
        <td>
          24956.45306025214
        </td>
        <td>
          20168.349073673246
        </td>
      </tr>
      <tr>
        <td>
          104
        </td>
        <td>
          31242.66532658836
        </td>
        <td>
          25222.46684523808
        </td>
        <td>
          20362.309908871102
        </td>
      </tr>
      <tr>
        <td>
          105
        </td>
        <td>
          31605.47416757539
        </td>
        <td>
          25489.184278966208
        </td>
        <td>
          20556.51852468123
        </td>
      </tr>
      <tr>
        <td>
          106
        </td>
        <td>
          31969.715142873778
        </td>
        <td>
          25756.607222699287
        </td>
        <td>
          20750.976749703143
        </td>
      </tr>
      <tr>
        <td>
          107
        </td>
        <td>
          32335.39122408935
        </td>
        <td>
          26024.737542623432
        </td>
        <td>
          20945.68639880458
        </td>
      </tr>
      <tr>
        <td>
          108
        </td>
        <td>
          32702.505409736717
        </td>
        <td>
          26293.577109861068
        </td>
        <td>
          21140.649273507057
        </td>
      </tr>
      <tr>
        <td>
          109
        </td>
        <td>
          33071.06072490415
        </td>
        <td>
          26563.127800484097
        </td>
        <td>
          21335.867162358627
        </td>
      </tr>
      <tr>
        <td>
          110
        </td>
        <td>
          33441.060220931475
        </td>
        <td>
          26833.391495526896
        </td>
        <td>
          21531.34184129523
        </td>
      </tr>
      <tr>
        <td>
          111
        </td>
        <td>
          33812.5069750978
        </td>
        <td>
          27104.37008099946
        </td>
        <td>
          21727.075073990556
        </td>
      </tr>
      <tr>
        <td>
          112
        </td>
        <td>
          34185.404090322096
        </td>
        <td>
          27376.065447900623
        </td>
        <td>
          21923.06861219486
        </td>
      </tr>
      <tr>
        <td>
          113
        </td>
        <td>
          34559.75469487316
        </td>
        <td>
          27648.479492231192
        </td>
        <td>
          22119.324196063557
        </td>
      </tr>
      <tr>
        <td>
          114
        </td>
        <td>
          34935.56194208985
        </td>
        <td>
          27921.614115007178
        </td>
        <td>
          22315.843554475592
        </td>
      </tr>
      <tr>
        <td>
          115
        </td>
        <td>
          35312.82901011154
        </td>
        <td>
          28195.471222273136
        </td>
        <td>
          22512.62840534229
        </td>
      </tr>
      <tr>
        <td>
          116
        </td>
        <td>
          35691.55910161759
        </td>
        <td>
          28470.05272511533
        </td>
        <td>
          22709.68045590677
        </td>
      </tr>
      <tr>
        <td>
          117
        </td>
        <td>
          36071.75544357597
        </td>
        <td>
          28745.360539675214
        </td>
        <td>
          22907.0014030346
        </td>
      </tr>
      <tr>
        <td>
          118
        </td>
        <td>
          36453.42128700068
        </td>
        <td>
          29021.396587162748
        </td>
        <td>
          23104.592933495827
        </td>
      </tr>
      <tr>
        <td>
          119
        </td>
        <td>
          36836.55990671705
        </td>
        <td>
          29298.162793869742
        </td>
        <td>
          23302.45672423848
        </td>
      </tr>
      <tr>
        <td>
          120
        </td>
        <td>
          37221.174601136234
        </td>
        <td>
          29575.661091183396
        </td>
        <td>
          23500.594442654598
        </td>
      </tr>
    </tbody>
  </table>
</section>
`;

exports[`Table Should render without props 1`] = `<section />`;
